update-local-deps.mjs

```diff
@@ -0,0 +1,31 @@
+import { readFile, writeFile } from "node:fs/promises";
```

Comment:
Maybe use relative package links and add a pre-package script that replaces relative versions with the correct ones?